dbt_project.yml

```diff
@@ -9,10 +9,10 @@ models:
     +schema: netsuite
     netsuite:
       intermediate:
-        +materialized: ephemeral
+        +materialized: table
     netsuite2:
       intermediate:
-        +materialized: ephemeral
+        +materialized: table
```

Comment:
was this just for testing?

Reply:
should change back if not, otherwise note the materialization change in the CHANGELOG